### DISCUSSION TITLE
Where to get wallpapers like the ones that are used in different themes

### DISCUSSION THREAD

**Discussion by Exoutia** [prasanthrangan/hyprdots#288] [General]
First of all thank all of you very much, who has made this config possible. I am loving every second I spending on my old computer thanks to you guys (may be too much) one thing that mesmerized me the most are the collection of wallpapers, THEY ARE SUPER AWESOME!.
I would love to know where you guys have found the wallpapers like these and can I get more of them.
I love the collection of catppucin mocha wallpaper most when I see them I feel very calm and serene.  
![231003_14h05m23s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/94464106/46c5d699-d523-4c87-a704-e1c1d19f532a)
![231003_14h07m08s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/94464106/a8aa4535-8116-4412-979a-5d188df75e43)

![231003_14h09m14s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/94464106/7a81bd4e-82fb-40e7-a371-f502641b13b1)

**Comment by prasanthrangan** [comment]
Hi, thank you so much for your support!
The wallpapers are mostly from reddit, that I collected overtime.... I don't remember the subreddits as I've not logged into reddit for a long time now... But let me check and share it!

Btw your nvim config (screenshot 3) looks on point!

**Reply by Exoutia** [reply]
Thank you so much I have given hours to this config 😅😅
